diff --git a/test/application_system_test_case.rb b/test/application_system_test_case.rb
index 27567bb9c..d0e528c58 100644
--- a/test/application_system_test_case.rb
+++ b/test/application_system_test_case.rb
@@ -18,7 +18,6 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 require File.expand_path('../test_helper', __FILE__)
-require 'webdrivers/chromedriver'
 
 class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
   DOWNLOADS_PATH = File.expand_path(File.join(Rails.root, 'tmp', 'downloads'))
@@ -42,9 +41,9 @@ class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
                       'download.prompt_for_download' => false,
                       'plugins.plugins_disabled' => ["Chrome PDF Viewer"]
                     }
-                  }
+                  },
                 )
-
+  options[:browser] = :remote
   driven_by(
     :selenium, using: :chrome, screen_size: [1024, 900],
     options: options
